app/helpers/adhoq/application_helper.rb

```diff
@@ -17,7 +17,7 @@ def schema_version
         ActiveRecord::SchemaMigration.maximum(:version)
       else
```

Comment:
hmm... `SchemaMigration` was introduced since 4.0 https://github.com/rails/rails/commit/67fba0cfa93feaa183d546de625e63cb16c56d7d.

If the adhoq need to support Rails 3.2, it should be reverted the commit.